payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncher.kt

```diff
@@ -55,7 +61,12 @@ class GooglePayLauncher internal constructor(
             GooglePayLauncherContract()
         ) {
             resultCallback.onResult(it)
-        }
+        },
+        AnalyticsRequestExecutor.Default(),
```

Comment:
nit: set this as default value of the internal constructor and avoid passing it here, as this value is always the same in production code